libs/api/ebpf_api.cpp

```diff
@@ -107,12 +109,13 @@ _get_ebpf_map_from_handle(ebpf_handle_t map_handle)
         map = it->second;
     }
 
-    return map;
+    EBPF_RETURN_POINTER(ebpf_map_t*, map);
```

Comment:
maybe not related to this PR, but why do we need to pass the pointer type to macro `EBPF_RETURN_POINTER`.

`EBPF_RETURN_POINTER` macro is only using the type to cast the pointer and then do a null check. Can we not do null check directly?